src/client/client.test.ts

```diff
@@ -1044,6 +1044,81 @@ describe('WebSocket URL Protocol Translation', () => {
   })
 })
 
+describe('WebSocket URL Protocol Translation with Query Parameters', () => {
+  const app = new Hono()
+  const route = app.get(
+    '/',
+    upgradeWebSocket((c) => ({
+      onMessage(event, ws) {
+        console.log(`Message from client: ${event.data}`)
```

Comment:
Can you remove this `console.log`?

Reply:
Thank you for your comment. It has been fixed!!